runtask.yml

```diff
@@ -9,10 +9,13 @@ input:
 
 environment:
     rpm:
+        - pdc-client
         - python2-aexpect
         - python2-avocado
         - python2-modulemd
+        - python2-requests
         - python2-enchant
+        - hunspell-en-US
```

Comment:
Just confirming--no need to change anything: Should this added line actually belong with PR #16?